psi4/src/psi4/libmints/eribase.cc

```diff
@@ -1779,6 +1779,104 @@ static size_t fill_primitive_data(prim_data *PrimQuartet, Fjt *fjt, const ShellP
     }
     return nprim;
 }
+static size_t fill_primitive_data_screen(prim_data *PrimQuartet, Fjt *fjt, ShellPairScreen sp12, ShellPairScreen sp34, int am,
+                                  bool sh1eqsh2, bool sh3eqsh4, int deriv_lvl) {
+    double zeta, eta, ooze, rho, poz, coef1, PQx, PQy, PQz, PQ2, Wx, Wy, Wz, o12, o34, T, *F;
+    double a1, a2, a3, a4;
+    int p12, p34, i;
+    size_t nprim = 0L;
+    for (p12 = 0; p12 < sp12.nonzeroPrimPairs.size(); ++p12) {
+        PrimPairScreen pp12 = sp12.nonzeroPrimPairs[p12]; 
```

Comment:
I'd make this `const PrimPairScreen& pp12 = ...` to avoid a copy.

Reply:
Thanks!